packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,389 @@
+          </View>
+          <View style={styles.centerContainer}>
+            <Text style={{ ...fontStyles.h2, textAlign: 'center', paddingVertical: 16 }}>
+              {t('dappsScreenBottomSheet.title', { dappName: dappSelected?.name })}
+            </Text>
+            <Text style={{ ...fontStyles.regular, textAlign: 'center', flex: 1 }}>
+              {t('dappsScreenBottomSheet.message')}
+            </Text>
+            <Button
+              style={styles.bottomSheetButton}
+              onPress={onPressNavigationButton}
+              text={t('dappsScreenBottomSheet.button', { dappName: dappSelected?.name })}
+            />
+          </View>
+        </View>
+      </BottomSheet>
+
+      <Dialog
+        title={t('dappsScreenHelpDialog.title')}
+        isVisible={isHelpDialogVisible}
+        secondaryActionText={t('dappsScreenHelpDialog.dismiss')}
+        secondaryActionPress={() => setHelpDialogVisible(false)}
+      >
+        {t('dappsScreenHelpDialog.message')}
+      </Dialog>
+
+      <ScrollView style={styles.scrollContainer}>
+        <View style={styles.descriptionContainer}>
+          <Text style={{ ...fontStyles.h1, flex: 1 }}>{t('dappsScreen.message')}</Text>
+          <View style={styles.descriptionImage}>
+            <DappsExplorerLogo />
+          </View>
+        </View>
+        <>
+          {loading && (
+            <View style={styles.centerContainer}>
+              <ActivityIndicator
+                style={styles.loadingIcon}
+                size="large"
+                color={colors.greenBrand}
+                testID="DAppExplorerScreen/loading"
+              />
+            </View>
+          )}
+          {!loading && error && (
+            <View style={{ ...styles.centerContainer, marginTop: 96 }}>
+              <Text style={fontStyles.regular}>{t('dappsScreen.errorMessage')}</Text>
+            </View>
+          )}
+          {!loading && !error && result && (
+            <SectionList
+              style={{ flex: 1, padding: 16 }}
+              sections={parseResultIntoSections(result)}
+              renderItem={({ item: category }) => (
+                <Dapp dapp={category} onPressDapp={onPressDapp} />
+              )}
+              keyExtractor={(item: Dapp) => `${item.categoryId}-${item.id}`}
+              renderSectionHeader={({
+                section,
+              }: {
+                section: SectionListData<any, SectionData>
+              }) => <CategoryHeader category={section.category} />}
+            />
+          )}
+        </>
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function parseResultIntoSections(categoriesWithDapps: CategoryWithDapps[]): SectionData[] {
+  return categoriesWithDapps.map((category) => ({
+    data: category.dapps,
+    category: category,
+  }))
+}
+
+function CategoryHeader({ category }: { category: CategoryWithDapps }) {
+  Logger.debug(TAG, `Render category ${JSON.stringify(category)}`)
```

Comment:
Is this log helpful? It looks like it would be a bit spammy since it's logged on every re-render, so it would keep logging the same thing over and over.

Reply:
I found it helpful while developing. But maybe we can delete them now.
It's true that it's logged on every re-render, however, there shouldn't be any of them, except the first one (after fetching the json file). So I also don't find it spammy.
On my opinion I will let them there. But happy to delete them if you still think we should.

Reply:
I removed them 😋

Reply:
Oh, sorry about that, it's true that it's not re-rendered that often, so I would have been fine if you preferred to keep them, but I guess it's fine either way :)